src/robusta/integrations/prometheus/models.py

```diff
@@ -133,8 +134,10 @@ def __get_alert_subject(self) -> FindingSubject:
         elif self.node:
             subject_type = FindingSubjectType.TYPE_NODE
             name = self.node.metadata.name
+        if self.node:
```

Comment:
why didn't you put it under the last `elif`? (elif self.node)?

Reply:
was unsure about the constructor for PrometheusKubernetesAlert and where node, pod, deployment are assigned,
If it was possible for multiple of them to be assigned I wanted to still get the node from the PrometheusKubernetesAlert but from what i see now it just inherits it from only one other event